WORKSPACE

```diff
@@ -1 +1,5 @@
 workspace(name = "bazel_skylib")
+
+load(":workspace.bzl", "bazel_skylib_workspace")
```

Comment:
Why push this into a bzl? Is it something consumers of skylib will have to do in their WORKSPACE or is this just local for the local test?  If it is local, why not just directly do the work here?

Reply:
ps - if it is something everyone else is going to have to do, then the README likely needs to also be updated to include the info.

Reply:
Yes, users of skylib have to call this macro to register the toolchains.

Reply:
Is it all users of skylib? Or just the ones that use `unittest.bzl`?

Reply:
After @katre 's comment (https://github.com/bazelbuild/bazel-skylib/pull/84#discussion_r238336792) I changed the approach. Now unittest.bzl defines the toolchain, and maprule will define its own toolchain, so that the platform-specific logic can be moved there.